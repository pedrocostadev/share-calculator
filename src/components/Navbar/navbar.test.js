import React from 'react';
import { render, cleanup } from '@testing-library/react-native';

import NavBar from './navbar';

describe('<NavBar />', () => {
  const mockedProps = {
    screenText: '1 + 2 = 3',
  };

  beforeEach(() => {
    cleanup();
  });

  test('should render', async () => {
    const { container } = render(<NavBar {...mockedProps} />);
    expect(container).toMatchSnapshot();
  });
});
