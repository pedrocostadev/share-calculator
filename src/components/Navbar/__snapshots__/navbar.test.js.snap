// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar /> should render 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View>
    <View
      style={
        Array [
          Object {
            "alignSelf": "flex-start",
            "borderColor": "red",
            "borderWidth": 1,
          },
          Object {
            "backgroundColor": "#000000",
            "height": 56,
          },
        ]
      }
    />
  </View>
</View>
`;
